pkg/joinserver/redis/registry.go

```diff
@@ -46,6 +53,17 @@ type DeviceRegistry struct {
 	Redis *ttnredis.Client
 }
 
+func (r *DeviceRegistry) getProvisionerUniqueID(dev *ttnpb.EndDevice) (string, error) {
```

Comment:
This does not even use the `r` - define it as just a function.
Also, can we avoid the `get` - e.g. `func uniqueProvisionerID(dev *ttnpb.EndDevice) (string, error)`?